comeet/UI/MainMenu/MainMenuViewModel.swift

```diff
@@ -26,14 +26,15 @@ class MainMenuViewModel : BaseViewModel {
     
     func fetchMeetings() {
         let start = ""
-        print(start)
+        print("LOADING MEETINGS....")
         let end = ""
         
         
         
         fetcher.getMeetings(organization: authenticator.getOrganization(), user: "", start: start, end: end) { [weak self] (meetings: [Meeting]?, error: Error?) in
             if let meetings = meetings {
                 
+                print("COMPLETE... PRESS RELOAD")
```

Comment:
The print statements won't be visible to the end-user.  What do we need to do to make the UI refresh happen automatically?